app/helpers/home_helper.rb

```diff
@@ -0,0 +1,2 @@
+module HomeHelper
```

Comment:
@apeksha2697  no need to commit helpers file, if you are not adding any piece of code there

Reply:
done